src/hackney.app.src

```diff
@@ -4,7 +4,7 @@
 {application, hackney,
   [
     {description, "simple HTTP client"},
-    {vsn, "1.15.0"},
+    {vsn, "1.15.1"},
```

Comment:
can you make  remove that change? I will take care of the release numbering.

Reply:
Done